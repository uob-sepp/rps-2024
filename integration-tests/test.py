import subprocess

output = subprocess.run(
    ["../app/build/install/app/bin/app", "-n=7", "-p1=AlwaysPaper"],
    stdout=subprocess.PIPE,
    text=True,
).stdout

with open("test.expected") as f:
    expected = f.read()

if output != expected:
    print(f"Expected:\n{expected}\nbut got:\n{output}")
    exit(1)
