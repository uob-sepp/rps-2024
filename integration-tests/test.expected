It's a draw!
It's a draw!
It's a draw!
It's a draw!
It's a draw!
